src/handler.ts

```diff
@@ -40,10 +40,10 @@ const handler = async (
       callback(null, 'Data processed successfully.');
     } catch (error) {
       if (error.body) {
-        logger.error(JSON.stringify(error.body));
+        logger.error(JSON.stringify(error.message));
         callback(
           null,
-          `Data processed unsuccessfully: ${JSON.stringify(error.body)}`,
```

Comment:
To me, it looks as if the intention here was to log `error.X` if it exists, otherwise log the entire error (whatever that might be), on line 49.

Therefore, shouldn't the if check now read `if (error.message) {` ?